.github/workflows/update-supported-distros.yml

```diff
@@ -0,0 +1,140 @@
+name: Update supported distros
+
+on:
+    schedule:
+        - cron: '30 6 * * 1' # Runs every Monday at 8:30 AM CEST (6:30 UTC)
+    workflow_dispatch: # Allow manual triggering
+
+jobs:
+    update_distros:
+        runs-on: ubuntu-latest
+
+        permissions:
+          contents: write
+          pull-requests: write
+          actions: write
+          issues: write
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+              with:
+                token: ${{ secrets.WORKFLOW_TOKEN }}
+              
+            - name: Set up Python
+              uses: actions/setup-python@v5
+              with:
+                python-version: '3.13' 
+
+            - name: Install dependencies
+              run: |
+                python -m pip install --upgrade pip
+                pip install requests pyyaml ruamel.yaml
```

Comment:
This needs to happen with devenv or you need to commit lockfile into the Action folder.